rsocket/RSocketServer.cpp

```diff
@@ -135,7 +135,6 @@ void RSocketServer::onRSocketSetup(
   auto responder = std::make_shared<ScheduledRSocketResponder>(
       std::move(connectionParams.responder), *eventBase);
   auto rs = std::make_shared<RSocketStateMachine>(
-      *eventBase,
```

Comment:
yep, that is a good suggestion. I saw that too. I will address that in another diff.